src/test/kotlin/com/amazon/ionpathextraction/PathExtractorTest.kt

```diff
@@ -14,7 +14,7 @@
 package com.amazon.ionpathextraction
 
 import com.amazon.ion.*
-import com.amazon.ion.system.IonSystemBuilder
+import com.amazon.ion.system.*
```

Comment:
Here, do we need it to be * instead of just required classes?

Reply:
IntelliJ did this for me. I assume it knows more about what is considered proper style in Kotlin than I do, as someone who has written almost no Kotlin.